pyproject.toml.jinja

```diff
@@ -36,6 +36,7 @@ dynamic = ["version"]
 
 [project.urls]
 "Bug Tracker" = "https://github.com/{{orgname}}/{{projectname}}/issues"
+"Documentation" = "https://{{orgname}}.github.io/{{projectname}}"
```

Comment:
This wouldn't work for Scipp itself. However, I am not sure the goal was to also make Scipp use the copier template? I think it was mostly for our python-only packages?

That said, we do have this thing to account for the Scipp case in the `docs/conf.py` file: https://github.com/scipp/copier_template/blob/main/docs/conf.py.jinja#L135

Reply:
I don't know to what extend the template works for scipp. But I updated the link.